blackbox-multi-scope/src/test/java/org/multi/crosscut/BeanCrossTest.java

```diff
@@ -1,24 +1,97 @@
 package org.multi.crosscut;
 
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
-import io.avaje.inject.BeanScope;
-import org.junit.jupiter.api.Test;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.Named;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.multi.many.BeanInMany;
+import org.multi.many.ManyModule;
+import org.multi.moda.BeanInModA;
+import org.multi.moda.ModAModule;
+import org.multi.modb.BOther;
 import org.multi.modb.BeanInModB;
+import org.multi.modb.ModBModule;
+import org.multi.modc.BeanInModC;
+import org.multi.modc.COther;
+import org.multi.modc.ModCModule;
+import org.multi.modd.BeanInModD;
+import org.multi.modd.ModDModule;
+import org.multi.mode.BeanInModE;
+import org.multi.mode.ModEModule;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import io.avaje.inject.BeanScope;
+import io.avaje.inject.spi.AvajeModule;
 
 class BeanCrossTest {
 
-  @Test
-  void bootstrap() {
+  private static Stream<Arguments> allModulesDefined() {
```

Comment:
IMO these tests are not what we ergonomically desire/need.

That is, we want to say "Give me all the modules that I need for CrossCutModule [and ideally give them in the correct order]" ... and use those to create a BeanScope.

These tests are more just ... we can order the given list of modules correctly.

Reply:
> These tests are more just ... we can order the given list of modules correctly.

do we have an existing test for that?